lefthook.yml

```
@@ -3,9 +3,11 @@ pre-commit:
   commands:
     linter:
       run: dart fix --apply lib && git add {staged_files}
+    riverpod_lint:
+      run: dart run custom_lint
```

[nits]
そういえば、この辺りのコマンドは fvm 経由で実行したほうが適切そうです
```suggestion
      run: fvm dart run custom_lint
```

---

おっしゃる通りですね...ここら辺は`fvm`経由で実行するように修正しました！
e332a50939ca3b62340dc8e77f9b9493fcae398f